components/views/works/Works.module.scss

```diff
@@ -1,113 +1,82 @@
 .WorksPageContent {
-    $banner-height: 34.5vh;
+    display: flex;
+    flex-direction: column;
+    background-color: var(--off-white);
 
-    & > img {
```

Comment:
Never see & > in css gotta
Look this up

Reply:
just means immediate children of the parent element. without the `>` it would mean any child img element in the sub tree of this component